src/Ryujinx/Ui/Windows/ControllerWindow.cs

```diff
@@ -905,7 +905,7 @@ private void HandleButtonPressed(ToggleButton button, bool forStick)
                     }
                     else if (pressedButton != "")
                     {
-                        button.Label = pressedButton;
+                        button.Label = pressedButton.ToString();
```

Comment:
```suggestion
                        button.Label = pressedButton;
```
Then the warning doesn't exist anymore.